build/AssemblyVersion.proj

```diff
@@ -3,6 +3,9 @@ Configures the version based on the build number. Assumes $AssemblyVersion has a
 depends on each project.
 -->
 <Project>
+	<PropertyGroup>
```

Comment:
Can you keep this change locally only?

Reply:
I think this would be a great addition to the code. I also have the same problem with the user name and can't build the code so I will not be able to contribute as I would like to do. Just my 2 cents.

Reply:
Sounds good. We can keep it.